src/views/open-api-docs-view/utils/find-default-version.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { OpenApiDocsVersionData } from '../types'
+import { sortDateVersionData } from './sort-date-version-data'
+
+/**
+ * Given an array of version data,
+ * Return the default version that should be shown.
+ *
+ * - If there is exactly one version, we treat it as the default version.
```

Comment:
Tried to be explicit about the heuristic here - hopefully it feels more intuitive now!